client/model.go

```diff
@@ -243,48 +243,58 @@ type TeamProjectAssignment struct {
 }
 
 type Policy struct {
-	Id                         string `json:"id"`
-	ProjectId                  string `json:"projectId"`
-	NumberOfEnvironments       int    `json:"numberOfEnvironments"`
-	NumberOfEnvironmentsTotal  int    `json:"numberOfEnvironmentsTotal"`
-	RequiresApprovalDefault    bool   `json:"requiresApprovalDefault"`
-	IncludeCostEstimation      bool   `json:"includeCostEstimation"`
-	SkipApplyWhenPlanIsEmpty   bool   `json:"skipApplyWhenPlanIsEmpty"`
-	DisableDestroyEnvironments bool   `json:"disableDestroyEnvironments"`
-	SkipRedundantDeployments   bool   `json:"skipRedundantDeployments"`
-	UpdatedBy                  string `json:"updatedBy"`
+	Id                          string `json:"id"`
+	ProjectId                   string `json:"projectId"`
+	NumberOfEnvironments        int    `json:"numberOfEnvironments"`
+	NumberOfEnvironmentsTotal   int    `json:"numberOfEnvironmentsTotal"`
+	RequiresApprovalDefault     bool   `json:"requiresApprovalDefault"`
+	IncludeCostEstimation       bool   `json:"includeCostEstimation"`
+	SkipApplyWhenPlanIsEmpty    bool   `json:"skipApplyWhenPlanIsEmpty"`
+	DisableDestroyEnvironments  bool   `json:"disableDestroyEnvironments"`
+	SkipRedundantDeployments    bool   `json:"skipRedundantDeployments"`
+	UpdatedBy                   string `json:"updatedBy"`
+	RunPullRequestPlanDefault   bool   `json:"runPullRequestPlanDefault"`
+	ContinuousDeploymentDefault bool   `json:"continuousDeploymentDefault"`
 }
 
 type PolicyUpdatePayload struct {
-	ProjectId                  string `json:"projectId"`
-	NumberOfEnvironments       int    `json:"numberOfEnvironments"`
-	NumberOfEnvironmentsTotal  int    `json:"numberOfEnvironmentsTotal"`
-	RequiresApprovalDefault    bool   `json:"requiresApprovalDefault"`
-	IncludeCostEstimation      bool   `json:"includeCostEstimation"`
-	SkipApplyWhenPlanIsEmpty   bool   `json:"skipApplyWhenPlanIsEmpty"`
-	DisableDestroyEnvironments bool   `json:"disableDestroyEnvironments"`
-	SkipRedundantDeployments   bool   `json:"skipRedundantDeployments"`
+	Id                          string `json:"id"`
+	ProjectId                   string `json:"projectId"`
+	NumberOfEnvironments        int    `json:"numberOfEnvironments"`
+	NumberOfEnvironmentsTotal   int    `json:"numberOfEnvironmentsTotal"`
+	RequiresApprovalDefault     bool   `json:"requiresApprovalDefault"`
+	IncludeCostEstimation       bool   `json:"includeCostEstimation"`
+	SkipApplyWhenPlanIsEmpty    bool   `json:"skipApplyWhenPlanIsEmpty"`
+	DisableDestroyEnvironments  bool   `json:"disableDestroyEnvironments"`
+	SkipRedundantDeployments    bool   `json:"skipRedundantDeployments"`
+	UpdatedBy                   string `json:"updatedBy"`
```

Comment:
Same here, this is a computed attribute and shouldn't be sent on payload...

Reply:
https://github.com/env0/terraform-provider-env0/pull/164/commits/5e1b64d65dbfb68b74b36fe6fae93fc49e076432